doc/examples/plot_entropy.py

```diff
@@ -3,17 +3,57 @@
 Entropy
 =======
 
-Image entropy is a quantity which is used to describe the amount of information
-coded in an image.
+In information theory, information entropy is the log-base-2 of the number of
+possible outcomes for a message.
+
+For an image, local entropy is related to the complexity contained in a given
+neighborhood, typically defined by a structuring element. A large number of
+various gray levels has a higher entropy than an homogeneous neighborhood.
```

Comment:
Either remove "large number of" or "various". Duplicate. "has" -> "have". Also, I don't understand the sentence in its current form. Maybe remove the sentence entirely?